### PR TITLE
Fix/docs fixes

### PR DESCRIPTION
**Pull request by pawartur**
## Description

Various docs fixes.

### CODE REVIEW

**Review by PaulLeCam — CHANGES_REQUESTED** [review]
Thanks, LGTM overall, just a few comments

File: website/docs/first-composite.mdx [hunk 1]
```diff
@@ -28,21 +28,21 @@ The following steps require a local Ceramic node to be running. If you do not al
   <TabItem value="npm">
 
 ```sh
-npx @ceramicnetwork/cli daemon --network=inmemory
+npx @ceramicnetwork/cli daemon
```
[the remark this answers is not in the document]
Comment (pawartur) [reply website/docs/first-composite.mdx:31]: `composedb composite:from-model kjzl6hvfrbw6ca7nidsnrv78x7r4xt0xki71nvwe4j5a3s9wgou8yu3aj8cz38e --ceramic-url=http://localhost:7007 --output=my-first-composite.json` doesn't work, if your node is not on clay.

[the remark this answers is not in the document]
Comment (PaulLeCam) [reply website/docs/first-composite.mdx:31]: Yes that's why it's explicitly set on https://composedb.js.org/docs/0.3.x/first-composite#importing-a-model but that doesn't mean all the other interactions need to be on Clay

[the remark this answers is not in the document]
Comment (pawartur) [reply website/docs/first-composite.mdx:31]: `composite:from-model` command doesn't take a `--network` flag.

[the remark this answers is not in the document]
Comment (PaulLeCam) [reply website/docs/first-composite.mdx:31]: Ah yes right, I still don't think it's a good idea for everyone to use Clay but I guess that's fine for now then 👍

**Review by PaulLeCam — APPROVED** [review]
Thanks!
I'm merging it so I can pull these changes in my PR.

**Pull request merged**